docs/en/status-communities/about-voting-to-change-community-visibility.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 685
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About voting to change community visibility
+
+All SNT holders can engage in [voting to include or exclude a community][vote-to make-a-community-private-or-public] from the Communities Discover page, and [voting to feature a community][vote-to-feature-a-community]. 
+
+Votes happen on the [community directory curation dApp][curate.status.app]. This is an independent, decentralized application deployed on the Optimism network and is separate from the Status app. Voting outcomes are solely determined by the voters, and Status has no control over these results.
+
+When a community is created, it's private by default, which is ideal for small, exclusive communities. However, if the community targets a broad audience, you might prefer to make it public.
+
+## The basics
+
+- Every SNT holder, not just community members or Status users, can participate in all voting steps.
+- Your voting power is determined by how many SNT tokens you stake.
```

Comment:
```suggestion
- Your voting power is determined by how many SNT tokens you hold.
```

Reply:
Using "stake" rather than "hold" because users can choose the number of SNT tokens for voting, it's not necessarily the amount they hold.